### ISSUE TITLE
current network does not support EIP-1559

### ISSUE THREAD

**Issue by Hesiodee** [MetaMask/metamask-extension#17944]
### Describe the bug

Hi, 
I am working on Aragon and Metamask, testnet Goerli. And since a few days I am blocked by this message while doing transactions:

Your transaction wasn't signed and no tokens were sent.
Error: “Invalid transaction params: params specify an EIP-1559 transaction but the current network does not support EIP-1559”

How can I solve this ploblem please? 
Thanks !

### Steps to reproduce

1.
Aragon 
metamask
Testnet Goerli
While transacting, message : Your transaction wasn't signed and no tokens were sent.

Error: “Invalid transaction params: params specify an EIP-1559 transaction but the current network does not support EIP-1559”

### Version

10.25.0

### Build type

None

### Browser

Chrome

### Operating system

Windows

**Comment by OfficialSupportt** [comment]
@Hesiodee We'd be happy to look into this for you 
kindly file a support ticket below 
https://forms.gle/LA7fG3N7FgvbzjPS9

**Comment by bschorchit** [comment]
Hey @Hesiodee, thanks for reporting this. I'm closing in favor of https://github.com/MetaMask/metamask-extension/issues/13862
We'll be investigating it in the coming week.
